src/Maker/MakeResetPassword.php

```diff
@@ -0,0 +1,277 @@
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        $io->title('Reset Password Bundle Requires:');
+        $requirements[] = '1) A user entity has been created.';
+        $requirements[] = '2) The user entity contains an email property with a getter method.';
+        $requirements[] = '3) A user repository exists for the user entity.'."\n";
+        $requirements[] = '<fg=yellow>bin/console make:user</> will generate the user entity and it\'s repository...'."\n";
+        $io->text($requirements);
+
+        // initialize arguments & commands that are internal (i.e. meant only to be asked)
+        $command
+            ->addArgument('from-email-address', InputArgument::REQUIRED)
+            ->addArgument('from-email-name', InputArgument::REQUIRED)
+            ->addArgument('controller-reset-success-redirect', InputArgument::REQUIRED)
+            ->addArgument('user-class')
+            ->addArgument('email-property-name')
+            ->addArgument('email-getter')
+            ->addArgument('password-setter')
+        ;
+
+        $io->section('- Email Templates -');
+        $emailText[] = 'Please answer the following questions that will be used to generate the email templates.';
+        $emailText[] = 'If you are unsure of what these answers should be, that\'s ok.';
+        $emailText[] = 'You can change these later after the templates have been generated.';
+        $io->text($emailText);
+
+        $input->setArgument('from-email-address', $io->ask(
+            'What email address will be used to send reset confirmations? I.e. admin@your-domain.com',
+            null,
+            [Validator::class, 'validateEmailAddress']
+        ));
+
+        $input->setArgument('from-email-name', $io->ask(
+            'What name will be associated with the email address used to send password reset confirmations? I.e. John Doe or Your Company, LLC.',
+            null,
+            [Validator::class, 'notBlank']
+        )
+        );
+
+        $interactiveSecurityHelper = new InteractiveSecurityHelper();
+
+        if (!$this->fileManager->fileExists($path = 'config/packages/security.yaml')) {
+            throw new RuntimeCommandException('The file "config/packages/security.yaml" does not exist. This command needs that file to accurately build the reset password form.');
+        }
+
+        $manipulator = new YamlSourceManipulator($this->fileManager->getFileContents($path));
+        $securityData = $manipulator->getData();
+        $providersData = $securityData['security']['providers'] ?? [];
+
+        $input->setArgument(
+            'user-class',
+            $userClass = $interactiveSecurityHelper->guessUserClass(
+                $io,
+                $providersData,
+                'Enter the User class that should be used with the "forgotten password" feature (e.g. <fg=yellow>App\\Entity\\User</>)'
+            )
+        );
+
+        $io->section('- Controller Template -');
```

Comment:
Let's take out the word "template" in these sections unless the code is *really* going to be used in a "template". In this case, the code will be used in the controller. Heck we could even put `ResetPasswordController` in the section title as we now this will be the class name.

Reply:
done